### PR TITLE
Completed javascript-and-json.

### PR DESCRIPTION
**Pull request by stefancarrera**
JSON

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  {
+    isbn: '0062564331',
+    title: 'Anansi Boys',
+    author: 'Neil Gainman'
+  }
+];
+
+console.log('array of books', books, typeof books);
+
+var JSONbooks = JSON.stringify(books);
```
Comment (Zhiwutian) [inline javascript-and-json/main.js:21]: This variable should not be capitalized

File: javascript-and-json/main.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+console.log('JSON books', JSONbooks, typeof JSONbooks);
+
+var student = '{"ID":"031941150", "Name":"Billy Baston"}';
+console.log('student', student, typeof student);
+
+var JSONstudent = JSON.parse(student);
```
Comment (Zhiwutian) [inline javascript-and-json/main.js:28]: same as above

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small updates needed!

**Comment by stefancarrera** [comment]
JSON

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**